src/test/java/teammates/test/cases/ui/InstructorFeedbackCopyActionTest.java

```diff
@@ -143,14 +146,11 @@ public void testExecuteAndPostProcess() throws Exception {
                      pageResult.getStatusMessage());
         
         expectedString =
-                "TEAMMATESLOG|||instructorFeedbackCopy|||instructorFeedbackCopy|||true|||Instructor|||"
-                + "Instructor 1 of Course 1|||idOfInstructor1OfCourse1|||instr1@course1.tmt|||"
-                + "Servlet Action Failure : \"\" is not acceptable to TEAMMATES as a/an feedback session name "
-                + "because it is empty. The value of a/an feedback session name should be no longer than "
-                + "38 characters. It should not be empty.|||/page/instructorFeedbackCopy";
+                teammatesLogMessage + "Servlet Action Failure : "
```

Comment:
Same as above. Basically I need to see the message as seen by the user. Looking at a bunch of constants doesn't give me the confidence that the user is seeing the correct message.